src/main/scala/tech/ant8e/sbt/i18n/SbtI18nPlugin.scala

```diff
@@ -39,9 +39,12 @@ object SbtI18nPlugin extends AutoPlugin {
             i18nBundlePackageName.value,
             i18nBreakOnMissingKeys.value
           ),
-          mappings in packageSrc ++= managedSources.value pair (Path
+          packageSrc / mappings ++= managedSources.value pair (Path
             .relativeTo(sourceManaged.value) | Path.flat),
-          sourceGenerators += generateI18NBundleTask.taskValue
+          sourceGenerators += generateI18NBundleTask.taskValue,
+          libraryDependencies ++= Seq(
```

Comment:
Not sure if this part will work. I'd like this dependency to be added to the project if the plugin is added.